packages/addons/service/jellyfin/source/bin/jellyfin.start

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+. /etc/profile
+oe_setup_addon service.jellyfin
+
+libs="${ADDON_DIR}/libs"
+ff="/storage/.kodi/addons/tools.ffmpeg-tools/bin"
+
+if [ ! -e "${libs}" ]; then
+  url="https://github.com/jellyfin/jellyfin/releases"
+  url_latest="${url}/latest"
```

Comment:
I really dislike the idea of always grepping the latest, we saw this failing at Chrome where a year after addon release Chrome  required a new dll version we didn't had yet at the repo. I would pin it to some version and maybe allow via addon to get the latest one ? Similar is at emby available.

From my pov the goal is to ship a known good version even if they is older.

Reply:
Ok - I'll look at some options. (I cloned the emby script pretty much as-is)

Reply:
All updates done except looking at how logic works with downloading the right Jellyfin version. Thinking move to distributing in the zip? instead of a download script?

Reply:
urgs 80mb, better just download it

Reply:
fixed - used the chrome-download as a base